### PR TITLE
doc: update changelog/readme notes for SAS content

### PR DESCRIPTION
**Pull request by scottdover**
**Summary**
This adds notes around SAS content changes

**Testing**
👀

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html). If you introduce breaking changes, please group them together in the "Changed" section using the **BREAKING:** prefix.
 
+## [Unreleased]
+
+### Added
+
+- Added SAS content navigator. You are now able to browse, edit, create, delete, and run files on a SAS server using a Viya connection.
```
Comment (scnwwu) [inline CHANGELOG.md:11]: Oh, just remembered our last meeting suggested to add issue or pr numbers to changelog

**Review by scnwwu — APPROVED** [review]
Looks good to me.

File: CHANGELOG.md (hunk 1 above)
Comment (jefurbee) [reply CHANGELOG.md]: Changes look good to me.

**Review by jefurbee — APPROVED** [review]
LGTM

**Pull request merged**